go/merkletreev2/position_test.go

```diff
@@ -0,0 +1,82 @@
+package merkletreev2
+
+import (
+	"math/big"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPositionEncoding(t *testing.T) {
```

Comment:
These unit test functions should use something like https://github.com/golang/go/wiki/TableDrivenTests